### PR TITLE
Fix: #17674

### LINKED ISSUE #17674: Missing lang attribute on the html start tag
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files#file_paths

### What information was incorrect, unhelpful, or incomplete?

The given HTML skeleton triggers a warning when passing it to the W3C validator.

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>My test page</title>
  </head>
  <body>
    <img src="" alt="My test image">
  </body>
</html>
```

> Warning: Consider adding a lang attribute to the html start tag to declare the language of this document.

### What did you expect to see?

The skeleton should include a `lang="en"` attribute to the start tag `html`, and the guide should explain that it is necessary to include one as a good practice. Below the resulting skeleton.

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>My test page</title>
  </head>
  <body>
    <img src="" alt="My test image">
  </body>
</html>

```

### Do you have any supporting links, references, or citations?

https://www.w3.org/International/techniques/authoring-html.en?open=language&open=textprocessing#textprocessing
https://www.w3.org/International/techniques/authoring-html.en?open=language&open=langvalues#langvalues

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/getting_started_with_the_web/dealing_with_files`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/getting_started_with_the_web/dealing_with_files/index.md
* Last commit: https://github.com/mdn/content/commit/4fa96a00be0c90a3cd291c4107f6faf6afd35799
* Document last modified: 2022-04-16T05:44:53.000Z

</details>

### PR DESCRIPTION
**Pull request by MrBrain295**
#### Summary

Add `lang` attribute.
#### Motivation

It needs that attribute.
#### Related issues

This fixes: #17674
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by VeryTastyTomato** [comment]
I don’t know about `en-US`, but the skeleton that has to be fixed isn’t used only in this location. For example it’s also used [here](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics#anatomy_of_an_html_document).

**Comment by VeryTastyTomato** [comment]
What I meant in the second part of my sentence about location, is that the skeleton isn’t used only in the previous webpage I linked in #17674 but also in other webpages in the guide, like [here](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/HTML_basics#anatomy_of_an_html_document).

Besides that, I have nothing against using `en-US`.

**Comment by MrBrain295** [comment]
Oh, ok.

**Comment by sideshowbarker** [comment]
@MrBrain295 Do you want to add changes for the other cases too in this PR? If not, we can go ahead and merge this change but leave https://github.com/mdn/content/issues/17674 open pending the other changes getting made by somebody in follow-up PRs.

File: files/en-us/learn/getting_started_with_the_web/dealing_with_files/index.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ To make files talk to one another, you have to provide a file path between them
 
     ```html
     <!DOCTYPE html>
-    <html>
+    <html lang="en-US>
```
Comment (Josh-Cena) [inline files/en-us/learn/getting_started_with_the_web/dealing_with_files/index.md:54]: Missing closing quote

**Comment by MrBrain295** [comment]
I fixed the other cases in the getting started with the web folder.

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**